client/src/related_items/Card.jsx

```diff
@@ -27,12 +35,32 @@ const Card = (props) => {
       })
   }, [productId])
 
+  useEffect(() => {
+    helpers.getProductReview(productId)
+      .then(res => {
+        setRating(res)
+      })
+  }, [productId])
+
+  const handleClick = (e) => {
```

Comment:
Where are you using this ?

Reply:
Im not yet, thats going to be used to handle the click on the star at the top of the card